source/_integrations/home_connect.markdown

```diff
@@ -31,8 +33,10 @@ The Home Connect integration allows users to integrate their home appliances sup
 The integration will add one Home Assistant device for each connected home appliance which will have the following entities:
 
 - A power switch
-- If the device has programs, switches for each of the individual programs will be added. Note that program options cannot be configured currently.
-- If the device has programs, a timestamp sensor for remaining time and a numeric sensor for the progress percentage.
+- If the device has programs:
+  - Switches for each of the individual programs will be added. Note that program options cannot be configured currently.
```

Comment:
Should I mark this as deprecated once https://github.com/home-assistant/core/pull/131641 gets merged?

Reply:
Deprecation are generally just removed from the documentation.